File: src/main/java/tk/avabin/tdg/beans/dtos/ItemDto.java
package tk.avabin.tdg.beans.dtos;

import lombok.Data;
import org.springframework.context.annotation.Scope;
import org.springframework.stereotype.Component;

import java.io.Serializable;

/**
 * Created by Avabin on 18.05.2017.
 */
@Component
@Data
@Scope("prototype")
public class ItemDto implements Serializable {
    private int id;
    private String name;
    private String desc;
    private long price;
}

File: src/main/kotlin/tk/avabin/tdg/beans/dtos/ItemDto.kt
package tk.avabin.tdg.beans.dtos

import org.springframework.context.annotation.Scope
import org.springframework.stereotype.Component
import java.io.Serializable

/**
 * Created by Avabin on 18.05.2017.
 */
@Component
@Scope("prototype")
class ItemDto(
        var id: Int = 0,
        var name: String = "",
        var desc: String = "",
        var price: Long = 0
) : Serializable